src/utils/telemetry/index.js

```diff
@@ -19,18 +19,18 @@ const send = function (type, payload) {
     type,
   })
 
-  if (DEBUG) {
-    console.log(`${type} call`, payload)
-    return Promise.resolve()
+  const args = [process.execPath, [requestFile, options]]
+  if (process.env.NETLIFY_TEST_TELEMETRY_WAIT === 'true') {
+    return execa(...args, {
```

Comment:
We set `NETLIFY_TEST_TELEMETRY_WAIT` in our tests since we want to wait for the telemetry request